README.md

```diff
@@ -1,70 +1,52 @@
-# Getting Started with Create React App
+# Leaderboard
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Additional description about the project and its features:
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React Js
+- HTML and CSS
+- JavaScript
+- [HTML & CSS3 & JavaScript Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+- Github and GithubFlow
+- Webpack
 
```

Comment:
- [ ] Also I recommend adding a project screenshot to give the first imagination to users about your app.